packages/Meshfree/test/tstNearestNeighborExodusGenerator.cpp

```diff
@@ -205,8 +210,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( ExodusProblemGenerator, one_to_one, Node )
                                                          num_src );
     auto src_field_copy = KOKKOS_LAMBDA( const size_t i )
     {
-        for ( size_t n = 0; n < field_dim; ++n )
-            src_field_device( i ) = src_field( i, n );
+        src_field_device( i ) = src_field( i, 0 );
     };
```

Comment:
Also you do not use the lambda anywhere else you don't need to make an object, you can directly declare it in the `parallel_for()`.